docker-compose.yml

```diff
@@ -156,18 +200,22 @@ services:
       - "9114:9114"
 
   keycloak:
-    image: quay.io/keycloak/keycloak:16.1.1
+    image: quay.io/keycloak/keycloak:18.0.2
     container_name: keycloak
-    command: ["-b", "0.0.0.0", "-Dkeycloak.migration.action=import", "-Dkeycloak.migration.provider=dir", "-Dkeycloak.migration.dir=/opt/jboss/keycloak/realm-config", "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING", "-Djboss.socket.binding.port-offset=1000", "-Dkeycloak.profile.feature.upload_scripts=enabled"]
+    hostname: keycloak
+    command: [ "start-dev", "--import-realm" ]
+    restart: unless-stopped
+    healthcheck:
+      test: [ "CMD", "curl", "--fail", "http://localhost:8080/health" ]
     ports:
       - 9080:9080
-      - 9443:9443
     environment:
-      KEYCLOAK_USER: admin
-      KEYCLOAK_PASSWORD: password
+      KC_HTTP_PORT: 9080
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: password
```

Comment:
so...    I go back and forth on the use of the `.env`...   But if we are introducing it, then maybe the `KEYCLOAK_ADMIN_PASSWORD` should be in there?

Reply:
Missed that comment, you're right, we could and probably should put other credentials in the `.env` file. And all the other credentials that we set. And we should use variables as much as possible to minimize the effort to update these things. Next commit incoming...

Reply:
i think variables for things that we really expect to change...    I don't mind hard coding things to make it as simple to understand as possible....    As we frequently say, you aren't expected to deploy this to PROD ;-)

Reply:
Though maybe in the future we rethink that???!!!!